src/Tribe/Widget/Manager.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Widgets manager for Tribe plugins.
+ *
+ * @package Tribe\Widget
+ * @since   TBD
+ */
+
+namespace Tribe\Widget;
+
+/**
+ * Class Widget Manager.
+ *
+ * @since  TBD
+ *
+ * @package Tribe\Widget
+ */
+class Manager {
+
+	/**
+	 * Current widgets.
+	 *
+	 * @since TBD
+	 *
+	 * @var array $current_widget An array containing the current widgets being executed.
+	 */
+	public $current_widget = [];
+
+	/**
+	 * Get the list of widgets available for handling.
+	 *
+	 * @since  TBD
+	 *
+	 * @return array An associative array of widgets in the shape `[ <slug> => <class> ]`
+	 */
+	public function get_registered_widgets() {
+		$widgets = [];
+
+		/**
+		 * Allow the registering of widgets into the our Tribe plugins.
+		 *
+		 * @since  TBD
+		 *
+		 * @var array An associative array of widgets in the shape `[ <slug> => <class> ]`.
+		 */
+		$widgets = apply_filters( 'tribe_widgets', $widgets );
+
+		return $widgets;
+	}
+
+	/**
+	 * Verifies if a given widget slug is registered for handling.
+	 *
+	 * @since  TBD
+	 *
+	 * @param  string $slug Which slug we are checking if is registered.
+	 *
+	 * @return bool Whether a widget is registered or not.
+	 */
+	public function is_widget_registered( $slug ) {
+		$registered_widgets = $this->get_registered_widgets();
+		return isset( $registered_widgets[ $slug ] );
+	}
+
+	/**
+	 * Verifies if a given widget class name is registered for handling.
+	 *
+	 * @since  TBD
+	 *
+	 * @param  string $class_name Which class name we are checking if is registered.
+	 *
+	 * @return bool Whether a widget is registered, by class.
+	 */
+	public function is_widget_registered_by_class( $class_name ) {
+		$registered_widgets = $this->get_registered_widgets();
```

Comment:
```suggestion
		$registered_widgets = $this->get_registered_widgets();
		
```

Add a blank line before the `return` statement.